mm2src/mm2_bin_lib/Cargo.toml

```diff
@@ -50,6 +50,7 @@ gstuff = { version = "0.7", features = ["nightly"] }
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 js-sys = { version = "0.3.27" }
 mm2_rpc = { path = "../mm2_rpc" }
+mm2_rpc_data = { path = "../mm2_rpc_data" }
```

Comment:
Sure it's redundant.
done

Reply:
It's not removed yet, also can you please don't force push changes while the PR in review stage? it makes reviewing the latest changes since the last review not easy.

Reply:
I missed it's usage in `mm2_wasm_lib`, what was confusing to me is that the wasm profile compiles without the `mm2_rpc_data` dependency, I guess it's because of the function that uses it, which is exposed as a wasm binding.